third-party/linshare/src/test/java/org/apache/james/linshare/LinshareBlobExportMechanismTest.java

```diff
@@ -115,4 +108,11 @@ void exportShouldFailWhenBlobDoesNotExist() {
                 .export())
             .isInstanceOf(BlobExportMechanism.BlobExportException.class);
     }
+
+    private void assertMessageReceivedByTheSmtpServer() {
```

Comment:
sorry, it's not what I meant.
My point is : you should wait for the mail to arrive with awaitility but you should then do assertion on the retrieved mail as a standalone step.

Reply:
well I still need to wait until I get those two mails from fakesmtp, so I still need to do some kind of assert in it for that at least, to be sure I got the mails. There is no api endpoint to check the number of mails so I need to check that the body of the answer has a size of 2 mails at least.

Then now after that I check what's in the second mail, as a standalone, like intended. 

If  it's still not good enough detail a bit more please thanks !